### DISCUSSION TITLE
Submission page closed before 11.59pm

### DISCUSSION THREAD

**Discussion by pobooo** [snap-stanford/ogb#395] [General]
Hi all,

It seems that the submission page of ogb-lsc has been closed before 11:59 pm PDT, 1st Nov.. It is 02:00 PDT AM now. Is there anything wrong with the submission time setting? Or is the actual DDL is 31st Oct. 11:59pm? 

Best regards,
Qiang

**Comment by weihua916** [comment]
Apologies, just re-opened. It should work now.

**Reply by pobooo** [reply]
Thanks for the fast reply.
